desktop/main.js

```diff
@@ -40,17 +40,6 @@ autoUpdater.logger.transports.file.level = 'info';
 // See https://www.npmjs.com/package/electron-log
 _.assign(console, log.functions);
 
-// setup Hot reload
-if (ELECTRON_ENVIRONMENT.isDev()) {
-    try {
-        require('electron-reloader')(module, {
-            watchRenderer: false,
-            ignore: [/^(desktop)/],
-        });
-        // eslint-disable-next-line no-empty
-    } catch {}
-}
-
```

Comment:
Unfortunately when `main.js` is transpiled by webpack `electron-reloader` does not work: https://github.com/sindresorhus/electron-reloader/issues/15

We transpile `main.js` even in dev so that we can use `CONFIG.js` 

I think `main.js` isn't updated that frequently so it's a tolerable tradeoff 
And we can still find a way to enable this - perhaps with the new configurations we can use `electron-webpack`

Reply:
Okay, I'm fine with this trade-off for now, and maybe we can make a follow-up issue to get hot-reload working with `desktop/main.js`.